packages/material-ui/src/InputLabel/InputLabel.js

```diff
@@ -195,6 +201,7 @@ InputLabel.propTypes = {
 
 InputLabel.defaultProps = {
   disableAnimation: false,
+  shrink: 'auto',
```

Comment:
@bytasv Correct. This would be a breaking change since the default behavior is changed. This should be removed.

Reply:
Noted, I will fix this. I had a problem running regression tests locally, will try to see what's up with that so that doesn't repeat in the future.

Reply:
No worries they're not working for me either on ubuntu. I spent a couple of hours with no luck. But that's one of the reasons why CI environments are useful.